.sonarlint/Plugin.json

```diff
@@ -0,0 +1,4 @@
+{
+    "sonarCloudOrganization": "sonarsource",
+    "projectKey": "org.sonarsource.dotnet:sonar-csharp"
```

Comment:
Where did this came from? The project key is `org.sonarsource.dotnet:sonar-dotnet`

https://sonarcloud.io/project/overview?id=org.sonarsource.dotnet%3Asonar-dotnet

Reply:
I updated the content.
I am not sure how to test it.

I started Rider in a cleaned version of the repo (no prior .idea folder), and I had a pop-up to bind the project to `sonaranalyzer-dotnet`.
Is the manual binding still required with this configuration?

Reply:
Yes, you should have a popup. You still need to provide token